azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/Constants.java

```diff
@@ -10,9 +10,9 @@
     public static final String APP_SETTING_WEBSITE_RUN_FROM_PACKAGE = "WEBSITE_RUN_FROM_PACKAGE";
     public static final String APP_SETTING_MACHINEKEY_DECRYPTION_KEY = "MACHINEKEY_DecryptionKey";
     public static final String APP_SETTING_WEBSITES_ENABLE_APP_SERVICE_STORAGE = "WEBSITES_ENABLE_APP_SERVICE_STORAGE";
-    public static final String APP_SETTING_WEBSITES_ENABLE_APP_SERVICE_STORAGE_VALUE = "false";
+    public static final String APP_SETTING_WEBSITES_ENABLE_APP_SERVICE_STORAGE_FALSE = "false";
```

Comment:
APP_SETTING_WEBSITES_DISABLE_APP_SERVICE_STORAGE

Reply:
Will fix it in later PR